iroh-net/src/discovery/dns.rs

```diff
@@ -0,0 +1,63 @@
+//! DNS node discovery for iroh-net
+
+use crate::{
+    discovery::{Discovery, DiscoveryItem},
+    MagicEndpoint, NodeId,
+};
+use anyhow::Result;
+use futures::{future::FutureExt, stream::BoxStream, StreamExt};
+
+use crate::dns;
+
+/// The n0 testing DNS node origin
+pub const N0_TESTDNS_NODE_ORIGIN: &str = "testdns.iroh.link";
```

Comment:
needs updating, @Arqu what's the state of using the new url?

Reply:
Changed to `dns.iroh.link`. Our server is running under both `testdns.iroh.link` and `dns.iroh.link` for now.